spark/src/test/java/io/ddf/spark/etl/TransformationHandlerTest.java

```diff
@@ -81,6 +81,34 @@ public void testTransformScaleStandard() throws DDFException {
     Assert.assertEquals(8, newddf1.getSummary().length);
   }
 
+  @Test
+  public void testFlattenArrayTypeColumn() throws DDFException {
+
+    ddf.setMutable(false);
```

Comment:
If we keep the mutable logic and can't write tests for both mutable = false and mutable = true, then mutable = true should have higher priority, since that's the only thing users can see now